.pre-commit-config.yaml

```diff
@@ -1,8 +1,29 @@
+fail_fast: true
+minimum_pre_commit_version: "2"
+
 repos:
   - repo: local
     hooks:
       - id: black
         name: black
         entry: make black
-        language: python
-        always_run: True
+        files: "fidesctl/"
```

Comment:
Since each hook is scoped to a fairly high-level directory, should we also add:
```yml
types: [file, python]
```
to each one? [Source](https://pre-commit.com/#config-types)

Reply:
will add